gtars/src/uniwig/mod.rs

```diff
@@ -430,10 +552,98 @@ pub fn uniwig_main(
                 }
             }
         }
+        }
+    );
+
+    let vec_strings = vec!["start", "core", "end"];
```

Comment:
It doesnt look like this ever changes (since its not `mut`). Can we just make it an array instead?